### PR TITLE
continue work on develop-tmsp from PR

### PR DESCRIPTION
**Pull request by benjaminbollen**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -28,13 +28,14 @@ dependencies:
     - sudo chmod +x /usr/bin/docker
     # Override host go
     - sudo service docker start
-    - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS quay.io
+    # NOTE [ben]: @afdudley, this is a temporary hack to enable PRs as of 0.12.0
+    # - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS quay.io
```
Comment (benjaminbollen) [inline circle.yml]: @AFDudley just to keep you in the loop; I've gone ahead and "done the stupid thing here", just to get it out of my way, but happy to work with you to get a clean decoupling of CI and CD in place.

**Comment by AFDudley** [comment]
oh... something just clicked, the proper fix should be a one-liner, so i'm looking into that.

**Comment by benjaminbollen** [comment]
@silasdavis alright, so this has been a bit messy, but at least it is at the state of "failing tests"; I have to put these test-passes and test-failures next to those currently in tmsp-unified to get some picture of how successful this merge has been.  Hope this makes some sense to you :)

File: erisdb/pipe/transactor.go [hunk 2]
```diff
@@ -212,92 +212,6 @@ func (this *transactor) TransactAndHold(privKey, address, data []byte, gasLimit,
 	return ret, rErr
 }
 
-func (this *transactor) Send(privKey, toAddress []byte, amount int64) (*Receipt, error) {
```
Comment (benjaminbollen) [inline erisdb/pipe/transactor.go]: @ebuchman @NodeGuy @silasdavis can someone briefly remind me why we are deprecating the Send and SendAndHold methods for eris-db?

Comment (NodeGuy) [inline erisdb/pipe/transactor.go]: Weird, I just implemented the Send and SendAndHold methods in eris-db.js to support them at Tyler's request.

**Comment by silasdavis** [comment]
can we include all the commits on 10th May from here: https://github.com/eris-ltd/eris-db/commits/tmsp-unified there are a number of fixes relating to renaming txs

**Comment by benjaminbollen** [comment]
absolutely, but they should be there.  I will now both check explicit commits are included and put the tests side-by-side

**Comment by silasdavis** [comment]
Merging this to try and get us all to a consistent base to work off, @benjaminbollen will open another to work through some issues, develop-tmsp should become our base to accept PRs into

File: erisdb/pipe/transactor.go (hunk 2 above)
Comment (benjaminbollen) [inline erisdb/pipe/transactor.go]: Interesting :)  I'll take that as a veto, and make sure we don't deprecate it unless more arguments are brought to the table :)

**Pull request merged**